src/extension.ts

```diff
@@ -40,6 +58,9 @@ export function activate(context: vscode.ExtensionContext) {
 
 		}).catch((err:any) => {
 			console.log("ERROR: " + err);
+			if(err === ECONNREFUSED) {
+				vscode.window.showErrorMessage('Kindly check the URL you have provided and ensure firewall has been disabled');
```

Comment:
Perhaps it would be an idea to write a simple method that validates the schema url input.
you can either use a library or write your own method.

```suggestion
				vscode.window.showErrorMessage('Connection refused.\n  1) Check the URL\n  2) Ensure firewall is configured correctly.');
```

Reply:
I was thinking of a simple regex check for the schema URL.